vuepress/docs/next/docs/reference/glossary.md

```diff
@@ -4,43 +4,47 @@
 
 | Term |  Description
 |:--|:--
-| Entando Application | An application running on the Entando Platform and available at a given URL |
-| Entando App Builder | The frontend of the Entando Platform. Hosts the Entando WCMS and provides the UI to interact with and configure Entando components. |
-| Entando App Engine | The core runtime engine for applications built with Entando App Builder. Provides the composition and rendering of the final application. |
-| Entando Bundle | A package built with Entando that contains one or more components |
+| Entando Application | An application built with the Entando Platform and available at a given URL |
+| Entando App Builder | The frontend of the Entando Platform that hosts the Entando WCMS and provides a feature-rich user interface to configure and interact with components, design and create pages, manage content and build applications |
+| Entando App Engine | The core runtime engine responsible for the primary out-of-the-box services required to develop Entando Applications; it exposes the Entando App Builder APIs, assembles and coordinates components within the Entando App Builder, manages the cluster infrastructure and provides the data access layer to persist the page and application design |
+| Entando Bundle | A packaged set of components and resources created on Entando |
 | Entando CLI (ent CLI) | Entando's core command line interface |
 | Entando Cloud Hub | A SaaS instance of an Entando Hub that contains a public and private collection of components |
-| Entando Component Generator (ECG) | An implementation of a JHipster blueprint that provides automation and templating to generate the components used to build an Entando Application |
-| Entando Component Manager (ECM) | Part of the ECR that coordinates the installation of the MFEs and MSs in an Entando Bundle |
+| Entando Cluster | The infrastructure Entando deploys on Kubernetes via the Entando Operator and controllers; each discrete constituent is associated with a custom resource definition, if applicable |
+| Entando Component Generator (ECG) | Entando's implementation of a JHipster blueprint that generates the components used to build an Entando Application via automation and templating |
+| Entando Component Manager (ECM) | Provides functionality to build and organize micro frontends and widgets from within the App Builder and manages the connections between an application and the installed microservices |
```

Comment:
i actually think "using" or "via" is better... but "from within" connotes an action taking place in a specific location whereas "inside" just indicates a location; this is really splitting hairs but the one somehow seems more suitable. moot because action >> location

Reply:
also - i think this was pulled directly from a published page